beacon-chain/core/blob/sidecar.go

```diff
@@ -1,19 +1,19 @@
 package blob
 
 import (
-	"github.com/ethereum/go-ethereum/crypto/kzg"
+	"github.com/protolambda/go-kzg/eth"
```

Comment:
it might be cool to import this as `kzg`

I think that's a more revealing name and lets the diff here be tiny and obvious

but it's okay like this